shell/platform/embedder/embedder.cc

```diff
@@ -1317,7 +1317,7 @@ FlutterEngineResult FlutterEngineInitialize(size_t version,
     for (int i = 0; i < args->dart_entrypoint_argc; ++i) {
       arguments[i] = std::string{args->dart_entrypoint_argv[i]};
     }
-    settings.dart_entrypoint_args = std::move(arguments);
+    run_configuration.SetEntrypointArgs(arguments);
```

Comment:
std::move(arguments) right?

Reply:
Yeah, It sounds fine to me. I also tweak the related code from 
``` c++
 void RunConfiguration::SetEntrypointArgs(const std::vector<std::string>& entrypoint_args) {
   entrypoint_args_ = entrypoint_args;
 }
```
to 
``` c++
 void RunConfiguration::SetEntrypointArgs(std::vector<std::string> entrypoint_args) {
   entrypoint_args_ = std::move(entrypoint_args);
 }
```